src/test/java/seedu/address/testutil/TypicalWorkLists.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalEquipments.ACHORVALECC;
+import static seedu.address.testutil.TypicalEquipments.BOB;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.EquipmentManager;
+//import seedu.address.model.equipment.Equipment;
+import seedu.address.model.equipment.WorkList;
+
+/**
+ * A utility class containing a list of {@code WorkList} objects to be used in tests.
+ */
+public class TypicalWorkLists {
+
+    public static final WorkList LISTA = new WorkList("2019-01-01", "ALICE");
```

Comment:
same for this line. alice change to achorvale CC

Reply:
Okay. I will change it right now. Thank you Rachel for your comments.